code/modules/mob/living/carbon/xenomorph/castes/Drone.dm

```diff
@@ -20,7 +20,7 @@
 	build_time_mult = BUILD_TIME_MULT_BUILDER
 
 	caste_desc = "A builder of hives. Only drones may evolve into Queens."
-	evolves_to = list(XENO_CASTE_QUEEN, XENO_CASTE_BURROWER, XENO_CASTE_CARRIER, XENO_CASTE_HIVELORD) //Add more here separated by commas
+	evolves_to = list(XENO_CASTE_RUNNER, XENO_CASTE_SENTINEL, XENO_CASTE_DEFENDER, XENO_CASTE_QUEEN, XENO_CASTE_BURROWER, XENO_CASTE_CARRIER, XENO_CASTE_HIVELORD) //Add more here separated by commas
 	deevolves_to = list("Larva")
```

Comment:
These additional castes then should have their deevolves_to setting changed from larva to drone then. Additionally, the cost of evolution to previous T1s then need to be compensated for the cost of evolving to drone. See `/datum/caste_datum/New()` for how this previously was handled.

Reply:
Nevermind the larva bit especially if we're tweaking the evolution times.

Reply:
whole system got changed, please check code again.